### PR TITLE
zh: add FAQ about configUpdateStrategy

### LINKED ISSUE #1579: add a FAQ about configUpdateStrategy
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

    modify `.spec.tidb.config`, the new config does not take effect

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close #1579

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/faq.md [hunk 1]
```diff
@@ -138,3 +138,7 @@ PD 和 TiKV 使用 [Raft 一致性算法](https://raft.github.io/)将存储的
 * PD、TiDB、TiKV、TiFlash 任一 Pod 状态不是 Ready
 
 要判断 TiDB 集群是否真正不可用，你可以尝试连接 TiDB。如果无法连接成功，说明 TiDB 集群真正不可用。
+
+## 修改某个组件的配置后，为什么新配置没有生效？
+
+默认设置下，配置更新后不会进行滚动更新，新的配置也就不会生效。需要设置 `spec.configUpdateStrategy: RollingUpdate`，开启配置自动更新特性。该字段说明参考 [configUpdateStrategy](configure-a-tidb-cluster.md#configupdatestrategy)。
```
Comment (ran-huang) [inline zh/faq.md:144]:
```suggestion
默认设置下，配置更新后不会进行滚动更新，新的配置也就不会生效。需要设置 `spec.configUpdateStrategy: RollingUpdate`，开启配置自动更新特性。关于该字段的说明，参考 [configUpdateStrategy](configure-a-tidb-cluster.md#configupdatestrategy)。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**